### PR TITLE
SLE15 addons' name is changed, update the full name

### PR DESCRIPTION
**Pull request by tinawang123**
SLE15 addons' name is changed, update the full name

- Related ticket: https://progress.opensuse.org/issues/57191
- Verification run: 
http://10.161.8.44/tests/638#step/check_upgraded_service/7

### CODE REVIEW

File: lib/registration.pm [hunk 1]
```diff
@@ -656,11 +656,11 @@ sub get_addon_fullname {
     my %product_list = (
         ha        => 'sle-ha',
         geo       => 'sle-ha-geo',
-        we        => 'sle-we',
+        we        => is_sle('15+') ? 'product-we' : 'sle-we',
         sdk       => is_sle('15+') ? 'sle-module-development-tools' : 'sle-sdk',
         ses       => 'ses',
         live      => is_sle('15+') ? 'sle-module-live-patching' : 'sle-live-patching',
-        asmm      => 'sle-module-adv-systems-management',
+        asmm      => is_sle('15+') ? 'sle-module-basesystem' : 'sle-module-adv-systems-management',
```
Comment (foursixnine) [inline lib/registration.pm]: is this compatible with SLE15SP0, SLE15SP1?

File: lib/registration.pm [hunk 2]
```diff
@@ -656,11 +656,11 @@ sub get_addon_fullname {
     my %product_list = (
         ha        => 'sle-ha',
         geo       => 'sle-ha-geo',
-        we        => 'sle-we',
+        we        => is_sle('15+') ? 'product-we' : 'sle-we',
```
Comment (foursixnine) [inline lib/registration.pm]: see below

**Comment by tinawang123** [comment]
I have checked sle15 and gotten the summary:
  *  Advanced Systems Management Module will be upgraded to Basesystem Module 15 ppc64le.                                                                │
  │ *  Containers module will be upgraded to Containers Module 15 ppc64le.                                                                                 │
  │ *  Desktop Applications Module 15 ppc64le will be installed.                                                                                           │
  │ *  Server Applications Module 15 ppc64le will be installed.                                                                                            │
  │ *  Toolchain Module will be upgraded to Development Tools Module 15 ppc64le.                                                                           │
  │ *  Legacy Module will be upgraded to Legacy Module 15 ppc64le.                                                                                         │
  │ *  Public Cloud Module will be upgraded to Public Cloud Module 15 ppc64le.                                                                             │
  │ *  Web and Scripting Module will be upgraded to Web and Scripting Module 15 ppc64le.                                                                   │
  │ *  SUSE Linux Enterprise Server 12 SP5 will be upgraded to SUSE Linux Enterprise Server 15 ppc64le.

File: lib/registration.pm (hunk 1 above)
Comment (tinawang123) [reply lib/registration.pm]: Yes

**Comment by tinawang123** [comment]
Same situation with we:
https://migration-smt.qa.suse.de/repo/SUSE/Products/SLE-Product-WE/
sle15, sle15sp1, sle15sp2

**Comment by tinawang123** [comment]
Verified run: 
https://openqa.suse.de/tests/3400147#step/check_upgraded_service/266

**Comment by tinawang123** [comment]
> thx, but this is maybe true for SLE15SP2+ not for 15 or 15SP1
> 
> have you ever tried this change against real released product ? Against real SCC ?

Yes, I used the real released product with SCC.

**Pull request merged**